src/components/Bullet.tsx

```diff
@@ -413,6 +414,7 @@ const Bullet = ({
     const isHolding = state.draggedSimplePath && head(state.draggedSimplePath) === head(simplePath)
     return isHolding || isDragging
   })
+  const value = useSelector(state => getThoughtById(state, thoughtId)?.value)
```

Comment:
It's better to move `isDivider` into the selector here. Then the component will only be re-rendered when the boolean value changes, not every time the thought value changes.